### PR TITLE
Close changelog for beta1 release

### PR DESCRIPTION
**Pull request by tsg**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -112,33 +167,14 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha5...master[Check the HEAD d
...
 
 - Add harvester_limit option {pull}2417[2417]
 
-*Winlogbeat*
-
-
-==== Deprecated
-
-*Affecting all Beats*
-- Topology map is deperecated. This applies to the settings: refresh_topology_freq, topology_expire, save_topology, host_topology, password_topology, db_topology
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: What happened to this line?

Comment (tsg) [inline CHANGELOG.asciidoc]: That looked like duplicate to me. Restored now.

**Comment by ruflin** [comment]
LGTM

**Comment by tsg** [comment]
Rebased to master to include also #2571.

**Pull request merged**